src/jsrs_parser.cc

```diff
@@ -456,8 +463,10 @@ MaybeLocal<Value> ParseString(Isolate*    isolate,
 
     if (begin[i] == '\\') {
       if (IsLineTerminatorSequence(begin + i + 1, &in_offset)) {
+        use_fallback(i - 1);
```

Comment:
We can move this line to the outer block and get rid of the analogous one in the `else` clause, can't we?